packages/hash/playwright/tests/entity-creation.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { loginUsingTempForm } from "./shared/login-using-temp-form";
+import { resetDb } from "./shared/reset-db";
+import { expect, Locator, Page, test } from "./shared/runtime";
+
+const getCellText = async (
+  canvasLocator: Locator,
+  colIndex: number,
+  rowIndex: number,
+) => {
+  const text = await canvasLocator
+    .getByTestId(`glide-cell-${colIndex}-${rowIndex}`)
+    .textContent();
+  return text;
+};
+
+const ROW_HEIGHT = 42;
+
+const clickOnValueCell = async (
+  page: Page,
+  canvasLocator: Locator,
+  colIndex: number,
+) => {
+  const canvasPos = await canvasLocator.boundingBox();
+
+  if (!canvasPos) {
+    throw new Error("canvasPos not found");
+  }
+
+  const cellY = canvasPos.y + 20 + ROW_HEIGHT * (colIndex + 1);
+  const cellX = canvasPos.x + 300;
+  await page.mouse.move(cellX, cellY);
+  await page.mouse.click(cellX, cellY);
+};
+
+test.beforeEach(async () => {
+  await resetDb();
+});
+
+const value1 = "John";
+const value2 = "Doe";
```

Comment:
Unlike in app code, string repetition in tests is encouraged. The closer to humans we are, the more likely the tests will react to real changes in product (which is what we want). We can keep `value1` and `value2`, but if we switch to reading and writing strings directly, the tests will read a bit better:

```diff
-await page.keyboard.type(value1);
+await page.keyboard.type("John");

-expect(cell1Text).toBe(value1);
+expect(cell1Text).toBe("John");

```

Reply:
Updated here https://github.com/hashintel/hash/pull/1790/commits/29548ceb30f92b07c05ef2c19cce6daf2ad59f94